atom/browser/mac/atom_application.mm

```diff
@@ -4,6 +4,7 @@
 
 #import "atom/browser/mac/atom_application.h"
 
+#import "atom/browser/mac/atom_application_delegate.h"
```

Comment:
Shouldn't mac includes be `#import`? The one above it is also `#import`.
I might have missed any guidelines though, will check. Thanks.

Reply:
In general we `#include`, the `#import` above and the other still hanging around should be replaced unless they point to mac libraries like `<AppKit/AppKit.h>` afaik

Reply:
`atom_application_delegate.h` is an Objective-C++ only header without header guards, and we should only use `#include` when the header has header guards, otherwise recursive includes may happen.

We should probably add header guards to all headers and use `#include` for consistency, but let's do it in another PR.